README.md

```diff
@@ -14,6 +14,8 @@ Copy the code in this repo to where your Terraform config will live. For example
 
 If your `aws` commands run via [aws-vault](https://github.com/99designs/aws-vault), you'll need to set the `--no-session` flag so the IAM operations can run without being MFA'd. If you're using the [Truss aws-vault-wrapper](https://github.com/trussworks/terraform-layout-example/blob/master/bin/aws-vault-wrapper) you can set the `AWS_VAULT_NO_SESSION` environment variable. If you don't do this you'll receive an `InvalidClientTokenId` error.
 
+If you are running the aws-vault-wrapper as noted above, you will want to run this script without `AWS_VAULT_NO_SESSION`. The role assumption between different profiles requires session behavior. Additionally, this script will attempt to run `aws s3 ls` before checking for bucket existence so that you can create a session token that may require an MFA handshake.
```

Comment:
Doesn't `AWS_VAULT_NO_SESSION` only need to exist if we are using root Access Keys?

Reply:
I wrote this because how I read it implies you should always run this with AWS_VAULT_NO_SESSION.  I think more editing is necessary.

Reply:
pushed a change. Could use another pass.